system/Filters/Filters.php

```diff
@@ -375,7 +375,7 @@ public function enableFilters(array $names, string $when = 'before')
     /**
      * Returns the arguments for a specified key, or all.
      *
-     * @return mixed
+     * @return array|bool|float|int|object|string|null
```

Comment:
An argument is `string`, and `$this->arguments` is `array<string, string>`.
Therefore it should be `array|string`.